common/ast/src/parser/token.rs

```diff
@@ -792,14 +792,6 @@ impl TokenKind {
             // | TokenKind::WINDOW
             | TokenKind::WITH
             | TokenKind::DATEADD
```

Comment:
Maybe it'd be better if we rename it to `DATE_ADD`?

Reply:
OK